### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 21

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#293] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152320496-ee822208-df35-427a-9411-1ac6a48ff178.png)

**Comment by LuginiAndrea** [comment]
Trattiamo gli intervalli come insiemi
E' riflessiva: un insieme è sempre uguale a se stesso
E' anti-simmetrica: $XRY, YRX \Longrightarrow X = Y, Y = X \Longrightarrow X = Y$
E' transitiva: $XRY, YRZ \Longrightarrow X = Y, Y = Z \Longrightarrow X = Z \Longrightarrow XRZ$

**Comment by Elia-Belli** [comment]
- **Riflessiva [Vero]**: per ipotesi se $[x,y] = [w,z] \Rightarrow [x,y]R[w,z]$, siccome $[x,y]=[x,y]$ allora è sempre vero $[x,y]R[x,y]\space \forall x,y \in \mathbb R$, quindi la relazione è riflessiva.

- **Antisimmetrica [Vero]**: una relazione è antisimmetrica se $[x,y]R[w,z] \land [w,z]R[x,y] \Rightarrow [x,y]=[w,z]$.
Se le relazioni esistono perchè $[x,y] = [w,z]$, è evidente che l'antisimmetria si mantenga.
Analizziamo il caso in cui sono in relazione perchè $y\< w \land z\< x$ e se sia possibile.
Per definizione di intervallo $x\< y \land w\< z$, ma se $y\< w \land z\< x$ si arriva a contraddizione $x\< y\< w\< z < x$, ovvero $x \< x$, da questo sappiamo che è impossibile che $[x,y]R[w,z]$ e $[w,z]R[x,y]$ siano vere contemporaneamente.
Siccome l'antisimmetria si applica solo se le 2 relazioni esistono contemporaneamente allora in questo caso basta verificare se si mantiene per intervalli uguali, il che è evidente, quindi la relazione è antisimmetrica.

- **Transitiva [Vero]**: bisogna analizzare i vasi casi per cui $[x,y]R[w,z]$ e $[w,z]R[j,k]$
 Se $[x,y] = [w,z] = [j,k] \Rightarrow [x,y] = [j,k]$, caso verificato.
Se $y \< w \land [w,z] = [j,k] \Rightarrow y\< j$, caso verificato.
Se $[x,y] = [w,z] \land z \< j \Rightarrow y \< j$, caso verificato.
Se $y\< w \land z\< j \Rightarrow y\< j$, caso verificato.
Quindi la relazione è transitiva.
